library/src/main/java/in/krharsh17/barview/BarView.java

```diff
@@ -57,6 +81,13 @@ void addBar(BarModel data) {
                 ViewGroup.LayoutParams.WRAP_CONTENT
         ));
 
+        barGroup.setOnTouchListener(new OnTouchListener() {
```

Comment:
As far as I know, this will get called on `ACTION_DOWN` events, which will also include holding over a bar while scrolling the screen. Has that scenario been tested?